### PR TITLE
Fix row column inconsistency

### PR DESCRIPTION
**Pull request by benthorner**
This removes a few bits of redundant HTML. See the commits for more info...

### CODE REVIEW

File: app/views/documents/edit.html.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   } do |f| %>
 
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds" id="title">
```
Comment (kevindew) [inline app/views/documents/edit.html.erb]: These ids are used to link from summary page issues to the form

File: app/views/tags/edit.html.erb [hunk 2]
```diff
@@ -4,27 +4,23 @@
...
 
-<%= form_tag tags_path(@edition.document),
-             data: {
-               "warn-before-unload": "true",
-               gtm: "confirm-edit-tags"
-             },
-             class: "app-c-contextual-guidance__form" do %>
-  <% @edition.document_type.tags.each do |tag_field| %>
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds" id="<%= tag_field.id %>">
```
Comment (kevindew) [inline app/views/tags/edit.html.erb]: We'll want to keep this id somewhere for linking from summary

**Comment by kevindew** [comment]
darn you just beat me with the merge

**Comment by benthorner** [comment]
Oops. Revert: https://github.com/alphagov/content-publisher/pull/1260

**Pull request merged**